tests/kafkatest/tests/streams/streams_cooperative_rebalance_upgrade_test.py

```diff
@@ -68,9 +67,8 @@ def __init__(self, test_context):
                                            acks=1)
 
     @cluster(num_nodes=8)
-    @matrix(upgrade_from_version=streams_eager_rebalance_upgrade_versions)
-    def test_upgrade_to_cooperative_rebalance(self, upgrade_from_version):
-        self.zookeeper.start()
+    @matrix(upgrade_from_version=streams_eager_rebalance_upgrade_versions, metadata_quorum=[quorum.combined_kraft])
```

Comment:
Given that each parameter has exactly on value, it seem we could remove them?

Reply:
Right now the `metadata_quorum` in the `@matrix` annotation is required by test_context in `quorum.py` to determine if its using ZK or KRaft (maybe it'll pick it up from the parameter itself?) The `upgrade_from_version` points to an array so it doesn't have one value.

Reply:
Required to indicate to use KRaft for metadata quorum.  Failing to do so without a `@matrix` annotation results in this error as the test will default to ZK without it.
```File "/opt/kafka-dev/tests/kafkatest/services/kafka/kafka.py", line 541, in security_config
    zk_sasl=self.zk.zk_sasl if self.quorum_info.using_zk else False, zk_tls=self.zk_client_secure,
AttributeError: 'NoneType' object has no attribute 'zk_sasl'
```